r-pkg/R/es_search.R

```diff
@@ -737,8 +737,8 @@ es_search <- function(es_host
 # [examples]
 # \dontrun{
 # #--- Example 1: Basic Usage ---#
-# .ConvertToSec('1h') # returns 60*60 = 3600
-# .ConvertToSec('15m') # returns 60*15 = 900
+# .ConvertToSec('1h') # returns 60 * 60 = 3600
+# .ConvertToSec('15m') # returns 60 * 15 = 900
```

Comment:
interesting, it lints comments too!

Reply:
ha it doesn't, but when I did find-in-all-files for `*6` these  showed up and I thought "eh might as well"